packages/wrangler/src/pages/build.tsx

```diff
@@ -81,6 +81,18 @@ export function Options(yargs: Argv) {
 				type: "boolean",
 				hidden: true,
 			},
+			local: {
+				describe: "Build for local development",
+				default: false,
+				type: "boolean",
+				hidden: true,
+			},
+			bindings: {
+				type: "string",
+				describe:
+					"Bindings used in Functions (used to register beta product shims)",
+				deprecated: true,
```

Comment:
Should this be `hidden` @GregBrimble?

Reply:
Yeah, maybe not a bad shout to hide it as well. I'll add that.